server/index.py

```diff
@@ -0,0 +1,11 @@
+"""Server."""
+from flask import Flask
+from mimic.text_generator import TextGenerator
+from mimic.text_generator_factory import TextGeneratorFactory
+app = Flask(__name__)
+
+
+@app.route('/<model>/<string_length>', methods=['GET'])
```

Comment:
I agree with @frankfka - `string_length` and so on might be better suited as part of a JSON body. However I think for `model`, instead of taking it as a parameter we can just create one endpoint for each model, so `/shakespeare_lstm`, etc. We can discuss this when you all get here today though.